todo_app/app.py

```diff
@@ -1,11 +1,16 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, request
 
+from todo_app.data.session_items import *
```

Comment:
Minor, but it's usually better to avoid wildcard imports (`import *` ) since it can easily lead to overriding names - sticking with explicit imports or importing the namespace (below) are preferred. [The Python docs say](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package):
> Although certain modules are designed to export only names that follow certain patterns when you use import *, it is still considered bad practice in production code.
E.g. we could import like:
```python
import todo_app.data.trello_client as trello_client
trello_client.do_a_thing()
```